docs/reference/configuration/keywords.rst

```diff
@@ -1869,7 +1869,8 @@ bjobs.
         ERT will do exponential sleeps, starting at 2 seconds, and the provided
         timeout is a maximum. Let the timeout be sums of series like 2+4+8+16+32+64
         in order to be explicit about the number of retries. Set to zero to disallow
-        flakyness.
+        flakyness, setting it to 2 will allow for one re-attempt, and 6 will give two
```

Comment:
This is quite cumbersome to understand to be honest. If instead they could specify `QUEUE_QUERY_MAX_ATTEMPTS` which internally would transfer to TIMEOUT; that would be much nice I think :)